.gitignore

```diff
@@ -10,4 +10,7 @@
 .env
 
 # Local dev-docs folder
-/dev-docs
\ No newline at end of file
+/dev-docs
+
+# output of `cargo xtask lint`
+node_modules
```

Comment:
this shows up on running `cargo xtask lint`, probably created by the thing that links GraphQL schemas in `rover-client`